Source/Core/Gen4/Searchers/WildSearcher4.cpp

```diff
@@ -586,6 +616,12 @@ std::vector<WildState> WildSearcher4::cuteCharmMethodK(WildState state, u32 seed
 {
     std::vector<WildState> states;
 
+    PokeRNG rng2(seed);
+    rng2.advance(4);
+    u8 heldItem = rng2.nextUShort() % 100;
+
+    state.setHeldItem(heldItem);
+
```

Comment:
Move to parent scope. Basically line 237right before the if/else.

Will reduce code duplication and the amount of times this gets called.

Reply:
Done